src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_KEY_WORD = "supply";
+    private static final String BUY_KEY_WORD = "buy";
+    private static final String DATA_SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File fileFrom = new File(fromFileName);
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileFrom))) {
+            String line = reader.readLine();
+            int supply = 0;
+            int buy = 0;
+            while (line != null) {
+                String[] lineData = line.split(DATA_SEPARATOR);
+                if (lineData[0].equals(SUPPLY_KEY_WORD)) {
+                    supply += Integer.parseInt(lineData[1]);
```

Comment:
let's make `0` and `1` constants as well

Reply:
done